ibis/omniscidb/tests/test_ddl.py

```diff
@@ -0,0 +1,75 @@
+""" Tests for geo spatial data types"""
+import pytest
+
+import ibis
+
+pymapd = pytest.importorskip('pymapd')
+pytestmark = pytest.mark.omniscidb
+
+
+@pytest.fixture
+def new_schema():
+    return ibis.schema([('a', 'float'), ('b', 'double'), ('c', 'int32')])
+
+
+# table tests
+
+
+def test_create_table_from_schema(con, new_schema):
+    t_name = 'ibis_test_create_table_from_schema_omniscidb'
+
+    con.drop_table(t_name, force=True)
+
+    con.create_table(t_name, schema=new_schema)
+
+    try:
+        t = con.table(t_name)
+
+        for k, i_type in t.schema().items():
+            assert new_schema[k] == i_type
+    finally:
+        con.drop_table(t_name)
+
+
+def test_rename_table(con, new_schema):
+    t_name = 'ibis_test_rename_table_omniscidb'
+    t_name_new = '{}_new'.format(t_name)
+
+    # setup
+    con.drop_table(t_name, force=True)
+    con.drop_table(t_name_new, force=True)
+    con.create_table(t_name, schema=new_schema)
+
+    try:
```

Comment:
can you make a fixture so these are cleaned up there instead of in each test function

Reply:
done! thanks! it looks much better!